src/app/app.py

```diff
@@ -19,29 +18,34 @@ def create_app():
     app.config.from_object(config)
     app.app_context().push()
 
-    # Initialise slack
-    slack_client = WebClient(token=config.BOT_TOKEN)
-    slack_events_adapter = SlackEventAdapter(config.SECRET, config.EVENT_PATH, app)
+    # Initialise bolt
+    bolt = App(
+        token=config.BOT_TOKEN,
+        signing_secret=config.SIGNING_SECRET
+    )
 
     # Initialise redis
     redis = RedisClient(app, config.REDIS_URL, config.REDIS_TOKEN)
 
     # Initialise logger
-    dictConfig(json.load(open(from_root('app', 'config', 'logs.json'))))
-    app.logger_name = 'waffle-bot'
+    dictConfig(json.load(open(from_root("app", "config", "logs.json"))))
+    app.logger_name = "waffle-bot"
 
     # Initialise messages
-    messages = Messages(from_root('resources', 'messages.properties'))
+    messages = Messages(from_root("resources", "messages.properties"))
 
-    # Register blueprint
-    app.register_blueprint(archbishop.construct_blueprint(slack_events_adapter, config, messages, redis))
+    # Initialise archbishop
+    app.register_blueprint(archbishop.construct_blueprint(bolt, config, messages, redis))
 
     return app
 
 
-# Create the Flask app
+# Create Flask app
 app = create_app()
 
-# Only get to here when running directly
+# Only use when running direct
```

Comment:
Could you elaborate for me further? 
Is this only used when running a Postman request?

Reply:
This is executed whenever the app runs (i.e from docker, IDE or render) but not when the tests are run (@haydende, is that your understanding too?)

`if __name__ == "__main__":` allows your app to run as a script but not when imported as a module
https://realpython.com/if-name-main-python/#in-short-it-allows-you-to-execute-code-when-the-file-runs-as-a-script-but-not-when-its-imported-as-a-module

Reply:
@jrsmth / @jrsmth-tier2 - That's correct. The tests will be importing the `create_app` function and won't be using these selection statements.